dist/robotframework/res/api_shell.keywords.txt

```diff
@@ -26,20 +26,21 @@ API Call Should Error
     [Arguments]         ${call}  @{args}  &{kwargs}
     API Call Expect     Error  ${call}  @{args}  &{kwargs}
 
-API Call Get Result As Integer
+API Get Last Result As Integer
```

Comment:
Why last and not just API Get Result As Integer

Reply:
to me it sounds more explicit because its not any result but that of the **last** API call. Maybe the whole keyword should be wrapped differently -